test/entity/multipleFieldsEntity.js

```diff
@@ -1,135 +1,163 @@
 const { entity } = require('../../src/entity')
 const { field } = require('../../src/field')
 const assert = require('assert')
+const { id } = require('../../src/id')
 
 describe('An entity', () => {
+  describe('with multiple fields', () => {
```

Comment:
for some reason it looks like a whole new test session was created in this PR, but looking at the merge, it looks like only a few lines were added or changed. the ideal is to try to keep the PRs clean, showing only what has changed.

Reply:
Oh my.. My prettier just changed the hole file, let me fix it.